source/scala.txt

```diff
@@ -46,39 +46,33 @@ for more information.
 Connect to MongoDB Atlas
 ------------------------
 
-.. include:: /includes/atlas-connect-blurb.rst
+You can use the following connection snippet to test your connection to
+your MongoDB deployment on Atlas:
 
-.. code-block:: scala
-
-   import org.mongodb.scala._
-
-   // ...
-
-   // Replace the uri string with your MongoDB deployment's connection string.
-   val uri: String = "mongodb+srv://<username>:<password>@<cluster-address>/test?retryWrites=true&w=majority"
-   System.setProperty("org.mongodb.async.type", "netty")
-   val client: MongoClient = MongoClient(uri)
-   val db: MongoDatabase = client.getDatabase("test")
+.. literalinclude:: /includes/connection-snippets/scram/scala-connection.scala
+   :language: scala
 
-.. include:: /includes/serverless-compatibility.rst
+This connection snippet uses the {+stable-api+} feature, which you can
+enable when using the Scala driver v4.3 and later to connect to MongoDB Server
+v5.0 and later. When you use this feature, you can update your driver or server without 
```

Comment:
Q: is this clearer as "update your driver or server _version_"? What do you think?

Reply:
I think either is fine. You can update your server or driver or refer to it as updating the version